doordeck-sdk/src/androidMain/kotlin/com/doordeck/multiplatform/sdk/model/data/LockOperations.android.kt

```diff
@@ -329,77 +345,86 @@ internal fun LockOperations.LocationRequirement.toBasicLocationRequirement(): Ba
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.UnlockBetween.toBasicUnlockBetween(): BasicUnlockBetween {
     return BasicUnlockBetween(
-        start = start,
-        end = end,
-        timezone = timezone,
-        days = days,
-        exceptions = exceptions
+        start = start.toLocalTimeString(),
+        end = end.toLocalTimeString(),
+        timezone = timezone.id,
+        days = days.map { KDayOfWeek.valueOf(it.name) }.toSet(),
+        exceptions = exceptions?.map { it.toLocalDateString() }
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.UnlockOperation.toBasicUnlockOperation(): BasicUnlockOperation {
     return BasicUnlockOperation(
         baseOperation = baseOperation.toBasicBaseOperation(),
-        directAccessEndpoints = directAccessEndpoints
+        directAccessEndpoints = directAccessEndpoints?.map { it.toString() }
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.ShareLockOperation.toBasicShareLockOperation(): BasicShareLockOperation {
     return BasicShareLockOperation(
         baseOperation = baseOperation.toBasicBaseOperation(),
         shareLock = shareLock.toBasicShareLock()
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.ShareLock.toBasicShareLock(): BasicShareLock {
     return BasicShareLock(
-        targetUserId = targetUserId,
+        targetUserId = targetUserId.toString(),
         targetUserRole = targetUserRole,
-        targetUserPublicKey = targetUserPublicKey,
-        start = start?.toLong(),
-        end = end?.toLong()
+        targetUserPublicKey = targetUserPublicKey.encoded,
+        start = start?.epochSecond,
+        end = end?.epochSecond
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.BatchShareLockOperation.toBasicBatchShareLockOperation(): BasicBatchShareLockOperation {
     return BasicBatchShareLockOperation(
         baseOperation = baseOperation.toBasicBaseOperation(),
         users = users.map { it.toBasicShareLock() }
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.RevokeAccessToLockOperation.toBasicRevokeAccessToLockOperation(): BasicRevokeAccessToLockOperation {
     return BasicRevokeAccessToLockOperation(
         baseOperation = baseOperation.toBasicBaseOperation(),
-        users = users
+        users = users.map { it.toString() }
     )
 }
 
+@JvmSynthetic
 internal fun LockOperations.UpdateSecureSettingUnlockDuration.toBasicUpdateSecureSettingUnlockDuration(): BasicUpdateSecureSettingUnlockDuration {
     return BasicUpdateSecureSettingUnlockDuration(
         baseOperation = baseOperation.toBasicBaseOperation(),
-        unlockDuration = unlockDuration
+        unlockDuration = unlockDuration.seconds.toInt() // toSeconds() -> Call requires API level 31 (current min is 26)
```

Comment:
Here, I replaced `toSeconds()` (which requires API level 31) with `seconds`, which already exists and appears to do the same thing.

Reply:
So long as it's not getting *just* the number of seconds in a duration!